components/hab/src/cli/hab/svc.rs

```diff
@@ -147,8 +146,12 @@ pub struct SharedLoad {
                 possible_values = &["strict", "relaxed"])]
     pub binding_mode:          habitat_sup_protocol::types::BindingMode,
     /// The interval in seconds on which to run health checks
+    // We would prefer to use `HealthCheckInterval`. However, `HealthCheckInterval` uses a map based
+    // serialization format. We want to allow the user to simply specify a `u64` to be consistent
+    // with the CLI, but we cannot change the serialization because the spec file depends on the map
+    // based format.
```

Comment:
We really should fix this :frowning:

Reply:
Agreed. How do we do it without breaking backwards compatibility? I guess support both formats when `Deserialize`ing a spec file.

Reply:
Yup, exactly.